Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -270,12 +270,16 @@ void UIApplicationBackgroundActivated(BackgroundActivatedEventArgs^ args) {
 
 void DoApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg) {
     auto uiElem = ref new Xaml::Controls::Grid();
-    auto rootFrame = ref new Xaml::Controls::Frame();
-    rootFrame->Content = uiElem;
 
     XamlCompositor::Initialize(uiElem, activationType);
 
-    if (activationType != ActivationTypeLibrary) {
+    if (activationType == ActivationTypeLibrary) {
+        // In library mode, presented UI should completely cover whatever's behind it
+        uiElem->Background = ref new Xaml::Media::SolidColorBrush(Colors::White);
```

Comment:
Maybe, but it could also look weird if the controls behind the popup are visible but not interactive. Maybe a dimming effect would be appropriate?

In any event, it's moot for the moment, since all we're using right now is the Safari view controller, which doesn't do animations and is completely opaque. For now I'll leave this as is.

---
In reply to: [88291699](https://github.com/Microsoft/WinObjC/pull/1393#discussion_r88291699) [](ancestors = 88291699)